js/templates/modals/orderDetail/fulfillOrder.html

```diff
@@ -51,7 +51,7 @@
     </div>
     <div class="col7">
       <% if (ob.errors['cryptocurrencyDelivery.transactionID']) print(ob.formErrorTmpl({ errors: ob.errors['cryptocurrencyDelivery.transactionID'] })) %>
-      <input type="text" class="clrBr clrSh2" name="cryptocurrencyDelivery.transactionID" id="fulfillOrderTransactionId" value="<%= ob.cryptocurrencyDelivery.transactionID %>" placeholder="<%= ob.polyT(`orderDetail.fulfillOrderTab.transactionIdPlaceholder`) %>" />
+      <input type="text" class="clrBr clrSh2" name="cryptocurrencyDelivery.transactionID" id="fulfillOrderTransactionId" value="<%= ob.cryptocurrencyDelivery.transactionID %>" placeholder="<%= ob.polyT(`orderDetail.fulfillOrderTab.transactionIdPlaceholder`) %>" maxLength="<%= ob.constraints.transactionIDLength %>" />
```

Comment:
@jashot7 Have you tested this maxlength attr? The doc I see have it as not camel case, so I'm not sure if the camel case will work.